docs/operator-guide/cd/customize-environment-deletion.md

```diff
@@ -0,0 +1,190 @@
+# Customize Application Deletion Procedure
+
+KubeRocketCI provides UI capabilities to fully operate the CI/CD flow, starting from building applications to deploying its new release. The operation flow is completely seamless for most of the users. And nevertheless, sometimes there might be cases that call for additional, more specific efforts. A certain part of these extra efforts might be allocated to performing a custom approach to the application resources deletion procedure.
+
+## Delete Versus Clean
+
+Custom application deletion is implemented via the **Clean** and **Delete** buttons:
+
+  ![Clean button](../../assets/operator-guide/edp-portal-clean-application.png "Clean button")
+
+As you can see, there are two buttons in the Environment list that can perform the environment resources deletion: **Delete** and **Clean**. To not mix up these two buttons, let's review both of them:
+
+  * **Delete** - Deletes the Argo CD application custom resource, thus deleting the deployment, ingress, and other resources that are deployed while deploying application.
+  * **Clean** - Runs the custom delete pipeline, has its own logic. This pipeline is designed to cover deletion of all the resources, including non-Kubernetes ones.
```

Comment:
*Runs the custom delete pipeline, has its own logic*
Runs delete pipeline with custom logic

Reply:
please also remove: "This pipeline is designed to cover deletion of all the resources, including non-Kubernetes ones."